test/e2e/case38_install_operator_test.go

```diff
@@ -1832,1045 +1780,1419 @@ var _ = Describe("Testing OperatorPolicy", Ordered, Label("supports-hosted"), fu
 			)
 		})
 	})
-	Describe("Testing general OperatorPolicy mustnothave behavior", Ordered, func() {
-		const (
-			opPolYAML      = "../resources/case38_operator_install/operator-policy-mustnothave.yaml"
-			opPolName      = "oppol-mustnothave"
-			subName        = "project-quay"
-			deploymentName = "quay-operator-tng"
-			catSrcName     = "operatorhubio-catalog"
-			catSrcNS       = "olm"
-		)
-
-		BeforeAll(func() {
-			preFunc()
-			KubectlTarget("delete", "crd", "--selector=olm.managed=true")
-
-			createObjWithParent(parentPolicyYAML, parentPolicyName,
-				opPolYAML, testNamespace, gvrPolicy, gvrOperatorPolicy)
-		})
 
-		It("Should be Compliant and report all the things are correctly missing", func() {
-			check(
-				opPolName,
-				false,
-				[]policyv1.RelatedObject{{
-					Object: policyv1.ObjectResource{
-						Kind:       "OperatorGroup",
-						APIVersion: "operators.coreos.com/v1",
-						Metadata: policyv1.ObjectMetadata{
-							Namespace: opPolTestNS,
-							Name:      "-",
-						},
-					},
-					Compliant: "Compliant",
-					Reason:    "Resource not found as expected",
-				}},
-				metav1.Condition{
-					Type:    "OperatorGroupCompliant",
-					Status:  metav1.ConditionTrue,
-					Reason:  "OperatorGroupNotPresent",
-					Message: "the OperatorGroup is not present",
-				},
-				`the OperatorGroup is not present`,
-			)
-			check(
-				opPolName,
-				false,
-				[]policyv1.RelatedObject{{
-					Object: policyv1.ObjectResource{
-						Kind:       "Subscription",
-						APIVersion: "operators.coreos.com/v1alpha1",
-						Metadata: policyv1.ObjectMetadata{
-							Namespace: opPolTestNS,
-							Name:      "project-quay",
-						},
-					},
-					Compliant: "Compliant",
-					Reason:    "Resource not found as expected",
-				}},
-				metav1.Condition{
-					Type:    "SubscriptionCompliant",
-					Status:  metav1.ConditionTrue,
-					Reason:  "SubscriptionNotPresent",
-					Message: "the Subscription is not present",
-				},
-				`the Subscription is not present`,
+	// These tests can run in an Ordered container as long as no other tests install the
+	// same operator at the same time. These tests will delete and recreate the operator's CRD.
+	// The relative order of the inner Describe containers does not matter.
+	Describe("Testing CRD behaviors", Ordered, func() {
+		Describe("Testing full installation behavior, including CRD reporting", Ordered, func() {
+			const (
+				opPolYAML      = "../resources/case38_operator_install/operator-policy-no-group-one-version.yaml"
+				deploymentName = "apicast-operator-controller-manager-v2"
+				crdName        = "apicasts.apps.3scale.net"
```

Comment:
Note: Anyone who add tests to this file needs to be very careful if they decide to create an APIcast or Authorino operator, as I've used those for cluster-scoped tests. As far as I know, the other operators in the tests can be reused in parallel